packages/components/link/utils.ts

```diff
@@ -111,3 +111,17 @@ export const onHover = (el: HTMLAnchorElement, cb: () => void) => {
     el.removeEventListener("mouseout", mouseOutHandler);
   };
 };
+
+/**
+ * Removes the WP URL.
+ *
+ * @param link - The link URL.
+ * @param wpURL - The WP URL.
+ *
+ * @returns The URL without the WP URL.
+ */
+export const removeWPUrl = (link: string, wpURL: string) => {
```

Comment:
Maybe it would be great to add some unit tests for this util? It would be easier to fix edge cases and it would make sure it's not messed up in future refactorings. What do you think @nicholasio?

Reply:
Yep, will add tests for this.

Reply:
Great. Thanks :)

Reply:
Done.